tests/fsharp/typecheck/sigs/neg95.fs

```diff
@@ -50,12 +50,3 @@ namespace Neg95B
 
     [<Struct>]
     type StructUnion = StructUnion of float * StructUnion 
-
-    [<Struct>]
```

Comment:
Just to understand, why is this removed?

Reply:
I moved this test to the new ComponentTests/MultiCaseUnionStructTypes that I added as part of this PR

Reply:
Ah right, sure